packages/angular/projects/ui-angular/src/lib/components/amplify-sign-in/amplify-sign-in.component.html

```diff
@@ -2,41 +2,64 @@
   Define default contents here
 -->
 <ng-template #signInButton>
-  <button data-amplify-button type="submit">{{ signInButtonText }}</button>
+  <button
+    class="amplify-button"
+    data-fullwidth="true"
+    data-variation="primary"
+    type="submit"
+  >
+    {{ signInButtonText }}
+  </button>
 </ng-template>
 
 <ng-template #signInForm>
-  <amplify-federated-sign-in></amplify-federated-sign-in>
   <form data-amplify-form (submit)="onSubmit($event)" (input)="onInput($event)">
-    <fieldset data-amplify-fieldset [disabled]="isPending">
+    <fieldset
+      class="amplify-flex"
+      style="flex-direction: column"
+      data-amplify-fieldset
+      [disabled]="isPending"
+    >
+      <h3 class="amplify-heading">{{ this.headerText }}</h3>
       <amplify-user-name-alias></amplify-user-name-alias>
       <amplify-form-input
         data-amplify-password
         name="password"
         type="password"
         autocomplete="current-password"
       ></amplify-form-input>
-      <button type="button" (click)="toResetPassword()">
+      <ng-container
+        [ngTemplateOutlet]="customComponents.signInButton || signInButton"
+        [ngTemplateOutletContext]="context"
+      >
+      </ng-container>
+      <button
+        type="button"
+        class="amplify-button"
+        data-fullwidth="false"
+        data-size="small"
+        data-variation="link"
+        (click)="toSignUp()"
+        style="font-weight: normal"
+      >
+        {{ createAccountText }}
+      </button>
+      <button
+        type="button"
+        class="amplify-button"
+        data-fullwidth="false"
+        data-size="small"
+        data-variation="link"
+        (click)="toResetPassword()"
+        style="font-weight: normal"
+      >
         {{ forgotPasswordText }}
       </button>
-      <div data-amplify-footer>
-        <div>
-          {{ noAccountText }}
-          <button type="button" (click)="toSignUp()">
-            {{ createAccountText }}
-          </button>
-        </div>
-        <ng-container
-          [ngTemplateOutlet]="customComponents.signInButton || signInButton"
-          [ngTemplateOutletContext]="context"
-        >
-        </ng-container>
-      </div>
+      <amplify-error *ngIf="remoteError">
```

Comment:
This is a nitpick but you deleted the styles for `amplify-error`. So it no longer appears as red text, it's now black for errors

Reply:
Ah, good catch! Fixed in https://github.com/aws-amplify/amplify-ui/pull/410/commits/00c6d9f477bee8706cc13fece3f313b5c7c233e0.

![image](https://user-images.githubusercontent.com/43682783/134431638-7d225754-2bce-4e3a-949c-02d7d3936645.png)